ES/ESdd041.xml

```diff
@@ -177,144 +180,151 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           <note> The note is written in a hand which seems posterior to the main<ref type="hand" corresp="h1"> Hand 1</ref>. 
+                             It mentions <persName ref="PRS8550sardaDe"><roleName type="title">King</roleName> Malak Sagad (i.e., Śarḍa Dǝngǝl</persName>, r. <date>1563–97</date>) 
+                              as well as <persName ref="PRS8276SablaWa"><roleName type="title">ʾǝtege</roleName>Sabla Wangel</persName>, <persName ref="PRS5112harbo"><roleName type="title">ʾazmāč</roleName>Ḥarbo</persName>,
+                              <persName ref="PRS7278Muhammad">Maḥammad</persName> and<persName ref="PRS10343Yeshaq"><roleName type="title"> ʾazmāč</roleName> Yǝsḥaq</persName>. </note>
                         </item>
                         <item xml:id="a2">
-                           <locus target="#104v">f. 104va-b</locus>
-                           <desc type="GuestText">List of readings from various biblical books.</desc>
-                           <q xml:lang="gez"/>
-                           
-                           <note> The note contains indications for readings: a name of a biblical
-                              book, often abbreviated, and the beginning of the passage to be read
-                              (with numerous contractions). The readings are possibly connected to
-                              the ritual of monastic profession. </note>
+                           <locus target="#104v"/>
+                           <desc type="Unclear">List of readings from various biblical books;  the note contains indications for readings: a name of a biblical
+                              book, often abbreviated, and the beginning of the passage to be read (with numerous contractions). The readings are possibly connected to
+                              the ritual of monastic profession.</desc>
                         </item>
-                        <item xml:id="e1">- The title <q xml:lang="ti">ናይ፡ ምኵስና፡ መጽሐፋ</q> (Tǝgrǝñña) is crudely written,
-                           in pen, on <locus target="#1r">1r</locus>.                        </item>
-                        <item xml:id="e2"> - The name of the scribe,
-                           Malkǝʾa Krǝstos is written in red in the middle of the bottom margin on
-                        <locus target="#3r">3r</locus>.                        </item>
-                        <item xml:id="e3"> - Omitted letters or words
-                           written interlineally in a secondary hand: e.g., <locus target="#18vb">18vb</locus>, 21rb, 23ra, 30ra, 32vb, 34rb, 49rb, 50va, 50va, 52ra,
-                        55va, 58rb, 63rb, 69vb, 77va.                        </item>
+                        
+                        <item xml:id="e1">
+                           <locus target="#1r"/> 
+                           <desc >The title<q xml:lang="ti"> ናይ፡ ምኵስና፡ መጽሐፋ፡</q> (Tǝgrǝñña) is crudely written,
```

Comment:
Space in <desc >, line 254; also before ናይ፡ I would not do a space, but I would have one after "The title"